docs/readme.md

```diff
@@ -1 +1,22 @@
-- placeholder -
+## Plugin Install Instructions
+
+
+### Supported IDEs
+
+
+The Flutter Plugin can be installed on following IntelliJ-based products:
+
+* IntelliJ 2016.2+ ([Ultimate or Community](https://www.jetbrains.com/idea/download/))
+
+
+### Installation
+
+The Flutter Plugin is hosted in our own plugin repository.  Before it can be installed, the repository needs to be added to the IDE.
+
+1. Open plugin preferences (**Preferences>Plugins**)
+1. Select **”Browse repositories…”**
+1. In the **Browse Repositories** dialog, click **Manage repositories…”**
+1. In the resulting **Custom Plugin Repositories** pop-up, click the **+** button and paste `https://flutter.github.io/flutter-intellij/alpha/updatePlugins.xml` into the **Repository URL:** field and click **OK**
+1. In the **Browse Repositories** dialog, select **`flutter-intellij`** from the list of plugins (be sure it is not being filtered out of your results view by specifying **Category: All** or limiting it to the Flutter repository in the pull-down by the search box) and click the green **Install** button
+1. Once downloaded, click the **Restart IntelliJ IDEA** button
+
```

Comment:
I suggest we add a small section on usage:

## Using the plugin

Please consult the [core user journeys wiki page](https://github.com/flutter/flutter-intellij/wiki/User-Journeys).